api/document/js/commands.ts

```diff
@@ -0,0 +1,26 @@
+import { Node } from 'prosemirror-model';
+import { TextSelection } from 'prosemirror-state';
+
+import { factory } from './schema';
+
+export function addParagraph(state, dispatch) {
```

Comment:
I wonder if it'd be useful to add a comment above this function explaining exactly what the function does, e.g. "Appends a paragraph to the closest valid point after the head of the current selection"?  Eh, just a thought I guess.

Reply:
I had that thought too, I think it's a good idea.